front/src/components/oauth2/actions.js

```diff
@@ -0,0 +1,112 @@
+import { RequestStatus } from '../../utils/consts';
+
+const actions = store => ({
+  updateIntegrationName(state, e) {
+    store.setState({
+      integrationName: e
+    });
+  },
+  updateClientId(state, e) {
+    store.setState({
+      clientId: e.target.value
+    });
+  },
+  updateSecretId(state, e) {
+    store.setState({
+      secretId: e.target.value
+    });
+  },
+  async getCurrentConfig(state) {
+    store.setState({
+      oauth2GetStatus: RequestStatus.Getting
+    });
+
+    const returnServiceId = await state.httpClient.get(`/api/v1/service/${state.integrationName}/getServiceId`);
```

Comment:
`getServiceId` as REST path?

Reply:
I use it to get client_id and secret store in table t_variable, I need it to send authorisation link to third party authentication page and when this party back to callbackUrl choose the good init method (normally oauth2 page can be used for all others oauth2 provider just by config, only initialisation and poll method must be implemented by a new gladys service)